### PR TITLE
CBO-405: Case details travel distance warning messages

### PR DESCRIPTION
**Pull request by cleargif**
#### What
To list a set of warnings to draw the caseworker's attention to the distance calculations on an expense

#### Ticket

[CBO-405](https://dsdmoj.atlassian.net/browse/CBO-405)

#### Why
More clarity for caseworkers

### CODE REVIEW

File: app/views/shared/summary/_expenses.html.haml [hunk 1]
```diff
@@ -1,3 +1,16 @@
+.error-summary{"aria-labelledby" => "error-summary-expenses", :role => "alert", :tabindex => "-1"}
+  %h1#error-summary-heading-example-1.heading-medium.error-summary-heading
+    There are errors with the travel expenses
+  %p The distance claimed by the provider is greater than the calculated distance for the following expenses:
```
Comment (jsugarman) [inline app/views/shared/summary/_expenses.html.haml]: unless we want a more specific message i think `greater` will have to be `different`

**Comment by joelgsamuel** [comment]
@cleargif I don't work on CCCD, could you please avoid tagging me in your PRs?

File: app/views/shared/summary/_expenses.html.haml (hunk 1 above)
Comment (cleargif) [reply app/views/shared/summary/_expenses.html.haml]: @jsugarman  - I will raise it in JIRA.. thought that as well when I read the line

**Comment by cleargif** [comment]
@joelgsamuel - apologies, was a miss type.

File: app/views/shared/summary/expenses/show.html.haml [hunk 2]
```diff
@@ -1,52 +1,12 @@
-%tr.no-border
+%tr.no-border{id: "expense_id_#{expense.id}"}
```
Comment (jsugarman) [inline app/views/shared/summary/expenses/show.html.haml]: remove, no linking added following design review so not needed

File: config/locales/en.yml [hunk 3]
```diff
@@ -1269,6 +1269,13 @@ en:
           destination: Destination
           view_map_link: View on a map
           hours: Hours
+        distance_warnings:
+          distance_warning_header:
```
Comment (jsugarman) [inline config/locales/en.yml]: rename to just `header`

File: config/locales/en.yml [hunk 4]
```diff
@@ -1269,6 +1269,13 @@ en:
...
+            one: There is an error with the travel expenses
+            other: There are %{count} errors with the travel expenses
+          distance_warning_body:
```
Comment (jsugarman) [inline config/locales/en.yml]: rename to just `body`

**Pull request merged**